sphinx/tutorial/rose/metadata.rst

```diff
@@ -152,6 +152,9 @@ Metadata Items
       ``application-tutorial`` app we wrote in the
       :ref:`Rose application practical <rose-applications-practical>`.
 
+   In order to use ``rose config-edit``, you may find it easier to do this
+   whole practical using Rose 2019.
+
```

Comment:
Not too sure about this - users at sites using the Cylc wrapper script should be able to use Rose 2019 config edit without doing anything that we need to tell them here.

```suggestion
    If you get the following message:
    
    .. code-block:: bash
    
       The Rose configuration editor has been removed.
       The old Rose 2019 GUI remains compatible with Rose 2 configurations.
    
    You should open a terminal where ``CYLC_VERSION=7`` if you wish to use Rose 2019 graphical utilities.
    These utilities are compatible with Rose 2 configurations.
```